examples/directml/llm/test_model.py

```diff
@@ -0,0 +1,101 @@
+from transformers import AutoTokenizer
```

Comment:
This can be removed (keep it in a share somewhere if you want to reuse it layer on)